ambry-store/src/main/java/com/github/ambry/store/BootstrapController.java

```diff
@@ -0,0 +1,203 @@
+package com.github.ambry.store;
+
+import com.github.ambry.clustermap.ClusterParticipant;
+import com.github.ambry.clustermap.PartitionId;
+import com.github.ambry.clustermap.PartitionStateChangeListener;
+import com.github.ambry.clustermap.ReplicaId;
+import com.github.ambry.clustermap.StateModelListenerType;
+import com.github.ambry.clustermap.StateTransitionException;
+import com.github.ambry.config.StoreConfig;
+import com.github.ambry.server.StoreManager;
+import java.io.IOException;
+import java.util.Map;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.github.ambry.clustermap.StateTransitionException.TransitionErrorCode.*;
+
+
+public class BootstrapController {
```

Comment:
This should go to a different or separate module. Will fit better with on a higher level module imo

Reply:
Discussed offline with @aga9900 

This class may not end up with enough code that needs a separate module of its own, imo. 
While, this could be placed in either ambry-server or ambry-store (current) module, we can probably take that call later.

Reply:
ack